src/sgx/poet_enclave_sgx/poet.cpp

```diff
@@ -0,0 +1,108 @@
+/*
```

Comment:
Remove commented code, update copyright year, fix indentation.

Reply:
Sure, will take care of updating copyright and code cleanup